### ISSUE TITLE
missing apt packages for ubuntu noble (24.04)

### ISSUE THREAD

**Issue by ser** [zonemaster/zonemaster#1263]
it seems that ubuntu noble packages require some work, as previous versions do not work:

```
The following packages have unmet dependencies:
 libzonemaster-ldns-perl : Depends: perlapi-5.36.0 but it is not installable
```

**Comment by matsduf** [comment]
@ser, thank you for reporting the issue. When checking in a Ubuntu 22.04 system there is no `perlapi` installed. Is `perl-base` installed on the system?

My system was not installed with the packages, instead I used the following installation instruction using CPAN. Would that work for you use case, at least as a work-around?

https://github.com/zonemaster/zonemaster/blob/master/docs/public/installation/zonemaster-engine.md#installation-from-cpan

**Comment by ser** [comment]
I have installed Debian Bookworm instead of temporary, while waiting for packages for Noble.

**Comment by matsduf** [comment]
I am not sure what packages you are referring to. The official packages, created by the Zonemaster Project, are found via the instruction on https://github.com/zonemaster/zonemaster/blob/master/docs/public/installation/zonemaster-engine.md#installation-from-pre-built-packages. Those packages are not distributed via the official Ubuntu channel. Our packages are shared between all versions of Ubuntu and Debian supported by Zonemaster.

@ser, did you get the issue when installing the official packages or some other packages?

**Comment by ser** [comment]
I'm sorry, I think there is some misunderstanding. 

1. Zonemaster Project does not provide packages for Ubuntu Noble 24.04. 
2. My attempt to install packages from 23.10 was unsuccessful as it looks that package `perlapi-5.36.0` which was available in 23.10 is not available in 24.04.

**Comment by matsduf** [comment]
>     1. Zonemaster Project does not provide packages for Ubuntu Noble 24.04.

It should be possible to use the packages for Ubuntu 24.04 following the instructions linked to above. I will test to verify.

Those are the only packages that we any support on. Other packages might be out of date. If installation with packages does not work, then installation from CPAN is possible. See link further up.

**Comment by marc-vanderwal** [comment]
Packages should now be available for Ubuntu 24.04 LTS. Any feedback is more than welcome. 🙂

**Comment by ser** [comment]
Thank you!
